cypress/support/commands/login.js

```diff
@@ -0,0 +1,24 @@
+const login = (username, password, OTP = null, secondPassword = null) => {
+    cy.visit('/');
+    cy.url().should('include', '/login');
+    cy.get('#username').type(username);
+    cy.get('#password').type(password);
+    cy.get('#login_btn').click();
+
+    if (OTP !== null) {
+        cy.task('generateOTP', OTP).then((token) => {
+            cy.get('.TwoFA-input').type(token);
+            cy.get('#login_btn_2fa').click();
```

Comment:
can you add the marks we talked about yesterday ?

Reply:
Yes, they will be added in the proper repository. Once we start using the new login, I will update the tests.